src/Core/Algorithms/Visualization/OsprayDataAlgorithm.h

```diff
@@ -45,16 +46,35 @@ namespace SCIRun
         ALGORITHM_PARAMETER_DECL(DefaultColorG);
         ALGORITHM_PARAMETER_DECL(DefaultColorB);
         ALGORITHM_PARAMETER_DECL(DefaultColorA);
+        ALGORITHM_PARAMETER_DECL(Radius);
+        ALGORITHM_PARAMETER_DECL(UseNormals);
+        
+        typedef std::pair<int,int> Edge;
+        typedef std::vector<Edge> EdgeVector;
+        typedef boost::adjacency_list<boost::vecS, boost::vecS, boost::bidirectionalS> DirectedGraph;
+        typedef boost::adjacency_list<boost::setS, boost::vecS, boost::undirectedS, boost::no_property, boost::property<boost::edge_color_t, boost::default_color_type> > UndirectedGraph;
+        typedef boost::graph_traits<DirectedGraph>::vertex_descriptor Vertex;
+        typedef boost::graph_traits<UndirectedGraph>::vertex_descriptor Vertex_u;
+        typedef std::map<int, int> ComponentMap;
 
         class SCISHARE OsprayDataAlgorithm : public AlgorithmBase
         {
         public:
           OsprayDataAlgorithm();
           virtual AlgorithmOutput run(const AlgorithmInput& input) const override;
         private:
```

Comment:
No need to change this yet, but we'll want to unit test this class heavily, so this should be protected or public.